client/web-sveltekit/src/routes/search/SearchResults.svelte

```diff
@@ -16,23 +17,32 @@
     import { mdiCloseOctagonOutline } from '@mdi/js'
     import type { Observable } from 'rxjs'
     import { tick } from 'svelte'
+    import { writable } from 'svelte/store'
+
+    import { limitHit } from '@sourcegraph/branded'
```

Comment:
I think you can import that from `$lib/branded` (and if not, it should be added there)
```suggestion
    import { limitHit } from '$lib/branded'
```

Reply:
Whoops, got caught by auto-import. Thanks!